src/components/dom/back.js

```diff
@@ -2,10 +2,8 @@ import Link from 'next/link'
 
 const BackButton = () => {
   return (
-    <Link href='/' as={`/`}>
-      <button className='absolute z-20 p-2 m-2 text-white focus:outline-none focus:ring'>
-        Previous page
-      </button>
+    <Link href='/' as={`/`} >
+      <a className='absolute z-20 p-2 m-2 text-white focus:outline-none focus:ring'>Previous page</a>
```

Comment:
replaced the button so it match better in terms of a11y and it doesn't affect the styling